uaaextras/tests.py

```diff
@@ -507,36 +508,26 @@ def test_post_change_password_good(self, render_template, uaac):
             c.post(
                 '/change-password',
                 data={
-                    'old_password': 'Taeng0ooqua8ahchohGaechohS8zahG9',
-                    'new_password': 'oZaighoo0bu2ofio7chaefe6mah8ooTe',
-                    'repeat_password': 'oZaighoo0bu2ofio7chaefe6mah8ooTe',
+                    'old_password': 'fdsa',
+                    'new_password': 'password',
+                    'repeat_password': 'password',
                     '_csrf_token': 'bar',
                 }
             )
 
-        uaac().change_password.assert_called_with(
-            'example',
-            'Taeng0ooqua8ahchohGaechohS8zahG9',
-            'oZaighoo0bu2ofio7chaefe6mah8ooTe'
-        )
+        render_template.assert_called_with('change_password.html')
+        flash.assert_called_with('This is a top-10 common password.')
+        assert not uaac().change_password.called
 
-    @patch('uaaextras.webapp.flash')
-    @patch('uaaextras.webapp.zxcvbn')
     @patch('uaaextras.webapp.UAAClient')
     @patch('uaaextras.webapp.render_template')
-    def test_post_change_password_bad_too_simple(self, render_template, uaac, zxcvbn, flash):
-        """When a POST request is made to /forgot-password
-           and the user exists, send_email is called
+    def test_post_change_password_bad_repeat(self, render_template, uaac):
```

Comment:
From the name of the test, I expected that we would test the behavior of the `repeat_password` field--meaning we would test the case where `new_password` and `repeat_password` are different. This actually tests the case where `old_password` and `new_password` aren't different. Could you rename the test to describe what it does?

Reply:
I addressed this in `a38a094` and yeah @timothy-spencer I was thinking the same thing when I mentioned `new` !== `repeat`.